regression-tests/12-rpl/04-rpl-large-network.csc

```diff
@@ -51,7 +51,7 @@ make sender-node.cooja TARGET=cooja</commands>
       <description>RPL root</description>
       <source>[CONTIKI_DIR]/regression-tests/12-rpl/code/root-node.c</source>
       <commands>make TARGET=cooja clean
-make root-node.cooja TARGET=cooja</commands>
+make root-node.cooja DEFINES=UIP_CONF_MAX_ROUTES=400 TARGET=cooja</commands>

```

Comment:
Ok! I'll revert that back to what it was! Will get back with more fixes for this PR soon (including reverting back to 8 here).